src/modules/0.0.23/iasql_functions/sql/after_install.sql

```diff
@@ -165,3 +165,80 @@ begin
   ]') as x(name text, signature text, description text, sample_usage text);
 end;
 $$;
+
+CREATE
+OR REPLACE FUNCTION query_cron (_sql TEXT) RETURNS TEXT LANGUAGE plpgsql SECURITY DEFINER AS $$
```

Comment:
Basically they need to be fixed function things. It was okay with the RPC stuff because invalid RPC function names don't do anything and all RPCs were *designed* to be called by users.

Reply:
good point and good catch! thanks!